pages/ecosystem.vue

```diff
@@ -243,13 +271,22 @@ const joinAction: Link = {
     margin-top: carbon.$spacing-07;
   }
 
-  &__search {
+  &__toolbar {
+    margin-top: carbon.$spacing-06;
+  }
+
+  &__search,
+  &__sort-dropdown {
     margin-top: carbon.$spacing-06;
 
     --cds-field-01: #{carbon.$cool-gray-10};
     --cds-field-04: #{carbon.$cool-gray-30};
   }
 
+  &__sort-dropdown {
```

Comment:
this will still need the color overrides:

```scss
&__sort-dropdown {
  --cds-text-01: #{carbon.$cool-gray-60};
  --cds-field-01: #{carbon.$cool-gray-10};
  --cds-field-04: #{carbon.$cool-gray-30};
}
```